sys/posix/include/semaphore.h

```diff
@@ -34,6 +34,9 @@
 extern "C" {
 #endif
 
+/**
+ * @brief A sema_t is a sem_t is a sema_t
```

Comment:
Yes, `sema_t` is the RIOT name (to avoid name clashes with the later mentioned APIs), `sem_t` is the [POSIX type](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/semaphore.h.html). The `sys_sem_t` type is the system wrapper type used by lwIP for their internal semaphore usage.